desc/coils.py

```diff
@@ -294,21 +296,37 @@ def _compute_position(self, params=None, grid=None, dx1=False, **kwargs):
             Grid of coordinates to evaluate at. Defaults to a Linear grid.
             If an integer, uses that many equally spaced points.
         dx1 : bool
-            If True, also return dx/ds for the curve.
+            If True, also return dx/ds for the curve. Cannot be used
+            with ``center=True``
+        center : bool
+            whether or not to compute just the center of the coil.
 
         Returns
         -------
         x : ndarray, shape(len(self),source_grid.num_nodes,3)
             Coil positions, in [R,phi,Z] or [X,Y,Z] coordinates.
+            2nd dimension will be size 1 if center=True, as only one
+            center per coil is needed.
         x_s : ndarray, shape(len(self),source_grid.num_nodes,3)
             Coil position derivatives, in [R,phi,Z] or [X,Y,Z] coordinates.
             Only returned if dx1=True.
 
         """
         kwargs.setdefault("basis", "xyz")
-        keys = ["x", "x_s"] if dx1 else ["x"]
+        if not center:
+            keys = ["x", "x_s"] if dx1 else ["x"]
+        else:  # just compute coil centers
+            keys = ["center"]
+            # does not make sense to compute d(center)_ds
+            errorif(dx1, ValueError, "``dx1=True`` cannot be used with ``center=True``")
         data = self.compute(keys, grid=grid, params=params, **kwargs)
-        x = jnp.transpose(jnp.atleast_3d(data["x"]), [2, 0, 1])  # shape=(1,num_nodes,3)
+        x = jnp.transpose(
+            jnp.atleast_3d(data[keys[0]]), [2, 0, 1]
+        )  # shape=(1,num_nodes,3)
+        if center:
+            # make sure only carrying around 1 center point per coil
+            x = x[:, 0, :]
+            x = x[:, None, :]
```

Comment:
Why is this needed? Shouldn't it work the same as the `center = False` case with `num_nodes = 1` (since the center is a single position)?